components/icu/src/lib.rs

```diff
@@ -173,6 +173,56 @@ pub mod decimal {
     pub use icu_decimal::*;
 }
 
+pub mod locale_canonicalizer {
+    //! This API provides functionality to canonicalize locale identifiers based
+    //! upon [`CLDR`] data.
+    //!
+    //! It currently supports the minimize and maximize likely subtags algorithms
+    //! as described in [`UTS 35`].
+    //!
+    //! # Examples
+    //!
+    //! ```
+    //! # #[cfg(feature = "provider_serde")] {
+    //! use icu_locale_canonicalizer::{CanonicalizationResult, LocaleCanonicalizer};
+    //! use icu_locid::Locale;
+    //!
+    //! let provider = icu_testdata::get_provider();
+    //! let lc = LocaleCanonicalizer::new(&provider).unwrap();
```

Comment:
@zbraniecki suggested replacing `.unwrap()` with `.expect("message")`

Reply:
Yes, sorry, I was addressing your feedback and didn't see Zibi's until after I had pushed my changes.